rtloader/two/two.cpp

```diff
@@ -432,6 +432,25 @@ char *Two::runCheck(RtLoaderPyObject *check)
     return ret_copy;
 }
 
+void Two::cancelCheck(RtLoaderPyObject *check)
```

Comment:
Out of the scope of the PR, but the implementation of `two.cpp` is very closed to `three.cpp`.